requiremnets.txt

```diff
@@ -0,0 +1,68 @@
+absl-py==2.1.0
+altair==5.3.0
+altgraph==0.17.4
```

Comment:
Filename is a typo, and it's best to move this in pcb-util/
I'll merge this PR first and send the PR for the above comment.